tensorboard/webapp/metrics/views/_common.scss

```diff
@@ -61,6 +61,10 @@ $metrics-min-card-height: 320px;
 @mixin metrics-card-controls {
   @include tb-theme-foreground-prop(color, secondary-text);
   white-space: nowrap;
+
+  // ::ng-deep button[mat-icon-button] {
+  //   font-size: 16px;
+  // }
```

Comment:
> Somewhere in the metrics code we (unsuccessfully) try to override the height of the pin icon to 18px (I see it in the css browser but it doesn't get applied). Can you remove that as part of this change?

Can you address this comment I left previously?

Reply:
Good catch on the pin icon size. I removed the failed overrides.